models/lin_rate.h

```diff
@@ -0,0 +1,120 @@
+/*
+ *  lin_rate.h
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef LIN_RATE_H
+#define LIN_RATE_H
+
+// Includes from models:
+#include "rate_neuron_ipn.h"
+#include "rate_neuron_ipn_impl.h"
+#include "rate_neuron_opn.h"
+#include "rate_neuron_opn_impl.h"
+
+
+namespace nest
+{
+/* BeginDocumentation
+Name: lin_rate - Linear rate model
+
+Description:
+
+ lin_rate is an implementation of a linear rate model with either
+ input (lin_rate_ipn) or output noise (lin_rate_opn) and gain function
+ Phi(h) = g * h.
+
+ The model supports connections to other rate models with either zero or
+ non-zero delay, and uses the secondary_event concept introduced with
+ the gap-junction framework.
+
+Parameters:
+
+ The following parameters can be set in the status dictionary.
+
+ rate                double - Rate (unitless)
+ tau                 double - Time constant in ms.
+ mean                double - Mean of Gaussian white noise.
+ std                 double - Standard deviation of Gaussian white noise.
+ g                   double - Gain parameter
+ linear summation    boolean - specifies type of non-linearity
```

Comment:
We would like to keep the name, however, we added an explanation in the notes of the documentation making the name more comprehensible.